### PR TITLE
Document private APIs in GridViewHeaderRowPresenter

### LINKED ISSUE #20789: APIScan: Document three APIs in GridviewHeaderRowPresenter

### PR DESCRIPTION
**Pull request by gewarren**
Fixes #20789. For APIScan compliance.

[Preview link](https://review.docs.microsoft.com/en-us/dotnet/framework/additional-apis/system.windows.controls.gridviewheaderrowpresenter.findheaderbycolumn?branch=pr-en-us-20788).

### CODE REVIEW

File: docs/framework/additional-apis/system.windows.controls.gridviewheaderrowpresenter.prepareheaderdrag.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+## Parameters
+
+`header` <xref:System.Windows.Controls.GridViewColumnHeader>\
+The column header to prepare for reordering.
+
+`pos` <xref:System.Windows.Point>\
+The position, relative to <xref:System.Windows.Controls.GridViewHeaderRowPresenter>, where the dragging starts.
+
+`relativePos` <xref:System.Windows.Point>\
+The position, relative to `header`, where the dragging starts.
```
Comment (gewarren) [inline docs/framework/additional-apis/system.windows.controls.gridviewheaderrowpresenter.prepareheaderdrag.md:37]: I wasn't completely sure on this parameter's description.

File: docs/framework/additional-apis/system.windows.controls.gridviewheaderrowpresenter.prepareheaderdrag.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+`cancelInvoke` <xref:System.Boolean>\
+`true` to cancel the reordering; otherwise, `false`.
```
Comment (gewarren) [inline docs/framework/additional-apis/system.windows.controls.gridviewheaderrowpresenter.prepareheaderdrag.md:40]: I wasn't completely sure on this parameter's description.

File: docs/framework/additional-apis/system.windows.controls.gridviewheaderrowpresenter.prepareheaderdrag.md (hunk 1 above)
Comment (SamBent) [reply docs/framework/additional-apis/system.windows.controls.gridviewheaderrowpresenter.prepareheaderdrag.md:37]: Good enough for private method.

File: docs/framework/additional-apis/system.windows.controls.gridviewheaderrowpresenter.prepareheaderdrag.md (hunk 2 above)
Comment (SamBent) [reply docs/framework/additional-apis/system.windows.controls.gridviewheaderrowpresenter.prepareheaderdrag.md:40]: Good enough.

**Review by SamBent — APPROVED** [review]

**Comment by gewarren** [comment]
@dotnet/docs Could one of you sign off on this so I can merge it? It's high priority.

**Review by tdykstra — APPROVED** [review]

**Comment by gewarren** [comment]
Thanks Sam and Tom!

**Pull request merged**